test/integration/resourcemanager/csrapprover/csrapprover_test.go

```diff
@@ -96,35 +94,52 @@ var _ = Describe("Kubelet Server CertificateSigningRequest Approver Controller t
 		csrData, err := certutil.MakeCSR(privateKey, certificateSubject, dnsNames, ips)
 		Expect(err).NotTo(HaveOccurred())
 		csr.Spec.Request = csrData
-
-		By("Create CertificateSigningRequest")
-		Expect(testClient.Create(ctx, csr)).To(Succeed())
-		log.Info("Created CertificateSigningRequest for test", "certificateSigningRequest", client.ObjectKeyFromObject(csr))
-
-		DeferCleanup(func() {
-			By("Delete CertificateSigningRequest")
-			Expect(client.IgnoreNotFound(testClient.Delete(ctx, csr))).To(Succeed())
-		})
 	})
 
-	Context("non kubelet server certificate", func() {
+	Context("unhandled CSR signer", func() {
```

Comment:
The context is called `unhandled CSR signer`, but the `csr.Spec.SignerName` is set to `certificatesv1.KubeAPIServerClientSignerName` which should now be actually handled, right? I found this a bit confusing.